src/indieweb_utils/indieauth/flask.py

```diff
@@ -0,0 +1,103 @@
+import requests
```

Comment:
The two indieauth functions seemed to be flask specific or part of an application specific workflow. I wasn't quite sure of the best way to handle it, so I put it in flask.py.

Reply:
I have already added some documentation for the callback handler: https://indieweb-utils.readthedocs.io/en/latest/usage.html#handle-an-indieauth-callback-request. But, it looks like we have not yet documented the other function. `is_authenticated()` is more Flask specific since it requires a session storage object. I shall create an issue to add documentation for that function.